Easydict/Swift/View/SettingView/Tabs/ServiceConfigurationView/ServiceConfigurationCells.swift

```diff
@@ -54,9 +61,22 @@ struct ServiceConfigurationInputCell: View {
     let textFieldTitleKey: LocalizedStringKey
     let placeholder: LocalizedStringKey
 
+    var limitLength: Int
+
     var body: some View {
         TextField(textFieldTitleKey, text: $value ?? "", prompt: Text(placeholder))
             .padding(10.0)
+            .onReceive(Just(value)) { _ in
+                limit(limitLength)
```

Comment:
This implementation doesn't work very well, it can display over the length when typing and then truncate.

I've tried the answers here [how-to-set-textfield-max-length](https://stackoverflow.com/questions/56476007/how-to-set-textfield-max-length) , but none of them work well, can you help look at this?

Reply:
I couldn’t come up with anything else, either. 

We have constraints on the display in settings and query window, maybe it's fine to not limit this place.

Reply:
Yes, there are limits elsewhere, so it doesn't make sense to have too long inputs here, and I want the user to know that there is a limit to the length of inputs here.

If there is no good solution, leave it like this.